app/imports/templates/components/nameStatus.js

```diff
@@ -84,30 +84,30 @@ Template['components_nameStatus'].onCreated(function() {
             timeout = setTimeout(function() {
               if (name === Session.get('searched')) {
                 var value = entry.mode == 'owned' ? Math.max(Number(web3.fromWei(entry.value.toFixed(), 'ether')), 0.01) : 0;
-
+                var hash = entry.hash.replace('0x','').slice(0,12);
                 console.log('upsert', name);
-                Names.upsert({name: name}, {$set: {
+                Names.upsert(hash, {$set: {
```

Comment:
This line actually does that: https://github.com/ethereum/ens-registrar-dapp/pull/156/files#diff-86a33f65c8b4f19761e06fa65020c197R124

Avsa will create this change on his other branch, so i'll close this